contracts/src/LightClient.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: MIT
```

Comment:
Does this need to be changed? I think we haven't yet decided on what license to use for our contracts.

Reply:
Ah yes, defender didn't allow unlicensed contracts so that's something we have to decide to be able to move forward with defender.

Reply:
This is the error that Openzeppelin Defender returns:
```Error: License type UNLICENSED is not a valid SPDX license identifier for block explorer verification. Specify a valid license type.```

The rationale from threads like these https://forum.openzeppelin.com/t/solidity-0-6-8-introduces-spdx-license-identifiers/2859 suggest that the inclusion of valid license idenfiers (https://spdx.org/licenses/) is needed for verification of contracts on websites like Etherscan.